src/components/GestureComponents.tsx

```diff
@@ -53,7 +49,8 @@ export const DrawerLayoutAndroid = createNativeWrapper<
 // @ts-ignore FIXME(TS) maybe this should be removed?
 DrawerLayoutAndroid.positions = { Left: 'left', Right: 'right' };
```

Comment:
Is this still necessary after defining the type of `DrawerLayoutAndroid` as `typeof RNDrawerLayoutAndroid & RNDrawerLayoutAndroid`?

Reply:
Probably not.